packages/common/src/utils/streaming.ts

```diff
@@ -1,8 +1,7 @@
-import { select } from 'typed-redux-saga'
-
 import { PremiumContentSignature, Track } from 'models'
 import { AudiusBackend, QueryParams } from 'services/index'
 import { premiumContentSelectors } from 'store/premium-content'
+import { select } from 'typed-redux-saga'
```

Comment:
What's up with these reorderings? Were the files previously failing lint?

Reply:
Seems like they are still failing lint.

Reply:
yeah, weird - my vscode did these re-orderings automatically on save. i'll see what's up